infrastructure/variables.tf

```diff
@@ -104,6 +104,12 @@ variable "sb_topic_names" {
   })
 }
 
+variable "fd_name" { # front-door-docs
+  description = "The name of the resource."
+  type        = string
+  default     = "appeals-bo-fd"
+}
```

Comment:
I think we should use `"${local.org}-fd-${local.service_name}-docs-${var.environment}"` as per the pattern elsewhere and Confluence docs.